tests/test_exporter_git.py

```diff
@@ -0,0 +1,228 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import logging
+import pathlib
+import shutil
+import subprocess
+import typing as t
+
+import conftest
+import pytest
+from docker.models import containers
+
+log = logging.getLogger(__file__)
+log.setLevel("DEBUG")
+
+pytestmark = pytest.mark.t4c_server
+
+GIT_REPO_ENTRYPOINT: str = "/test-project/test-project.aird"
+
+SUBPROCESS_DEFAULT_ARGS: dict[str, t.Any] = {
+    "check": True,
+    "text": True,
+    "capture_output": True,
+}
+
+
+@pytest.fixture(name="prepare_local_git_server")
+def fixture_prepare_local_git_server(git_port: str, tmp_path: pathlib.Path):
+    checkout_git_repository(git_port, tmp_path)
+    copy_test_project_into_git_repo(tmp_path)
+    commit_and_push_git_repo(tmp_path)
+    yield
+
+
+@pytest.fixture(name="t4c_exporter_git_environment")
+def fixture_t4c_exporter_git_environment(
+    git_general_environment: dict[str, str],
+    t4c_exporter_general_environment: dict[str, str],
+) -> dict[str, str]:
+
+    return (
+        t4c_exporter_general_environment
+        | git_general_environment
+        | {"GIT_REPO_ENTRYPOINT": GIT_REPO_ENTRYPOINT}
+    )
+
+
+# We should just use @pytest.mark.usefixtures("perpare_local_git_server")
+# here, but right now there is a bug/issue that it doesn't work with
+# @pytest.fixture (https://github.com/pytest-dev/pytest/issues/3664)
+# So for now we have to keep it as an actual argument
+@pytest.fixture(name="t4c_exporter_git_parametrized_container")
+def fixture_t4c_exporter_git_parametrized_container(
+    t4c_exporter_git_environment: dict[str, str],
+    t4c_server_ports: tuple[str, str, str],
+    prepare_local_git_server: None,  # pylint: disable=unused-argument
+    request,
+) -> containers.Container:
+    if conftest.is_capella_6_x_x():
+        assert not conftest.get_projects_of_t4c_repository(t4c_server_ports[1])
```

Comment:
No, currently there is also the `test_export_model_5_x_x_unhappy` that simply checks whether the exporter container throws an error when run with an unsupported capella version. Therefore, this check is needed, since for capella <6.0.0 line 63 would already throw an error.